packages/manager/apps/nasha-react/src/pages/details/[serviceName]/informations.tsx

```diff
@@ -30,32 +60,107 @@ function Informations(props: { serviceName: string }) {
           'renameService',
           {
             serviceName,
-            data: { customName: serviceRename },
+            data: { customName: renameNasha },
           },
         ],
       });
 
-      // handle successful response here
-      // eslint-disable-next-line
-      console.log('Service renamed:', response);
+      console.info('Service renamed:', response);
+      setRedirectToSuccess(true);
     } catch (error) {
-      // handle error here
-      // eslint-disable-next-line
-      console.error('Error renaming service:', error);
+      setRedirectToError(true);
     }
   };
 
   return (
     <>
-      <ul>{JSON.stringify(data)}</ul>
-      <ul>
-        <input
-          type="text"
-          value={serviceRename}
-          onChange={(e) => setServiceRename(e.target.value)}
-        />
-        <button onClick={handleClickRename}>Modifier</button>
-      </ul>
+      <div className="element">
+        <OsdsText
+          level={OdsThemeTypographyLevel.subheading}
+          color={OdsThemeColorIntent.text}
+        >
+          {t('nasha_dashboard_information_name')}
+        </OsdsText>
+        <div>
+          <OsdsText color={OdsThemeColorIntent.text}>
+            {JSON.parse(JSON.stringify(data.serviceName))}
+          </OsdsText>
+          <div className="chipDashboard">
+            <button>
+              <OsdsLink color={OdsThemeColorIntent.primary}>
+                <OsdsIcon
+                  name={OdsIconName.ELLIPSIS}
+                  size={OdsIconSize.xxs}
+                  color={OdsThemeColorIntent.primary}
+                  onClick={handleClickRename}
```

Comment:
Why is the on click is on the icon and not on the button or the link ?
By the way, there is an option to make OsdsLink look like a button I think. Why do we wrap OsdsLink in a button ?